indico/modules/categories/controllers/display.py

```diff
@@ -118,18 +118,21 @@ def _count_users(self):
         return User.find(is_deleted=False, is_pending=False).count()
 
 
-def _serialize_category(category, with_path=False):
+def _serialize_category(category, with_path=False, with_favorite=False):
     data = {
         'id': category.id,
         'title': category.title,
         'is_protected': category.is_protected,
-        'category_count': category.deep_children_count,
-        'event_count': category.deep_events_count,
-        'can_access': category.can_access(session.user)
+        'event_count': len(category.events),
```

Comment:
Just added `category.event_count` in 89494cd - simply undefer it and you'll be able to use it without query spam